app/templates/components/demographic-change-table-row.hbs

```diff
@@ -1,18 +1,95 @@
-  <td>{{title}}</td>
-  <td>&nbsp;</td>
-  <td>&nbsp;</td>
-  <td>&nbsp;</td>
-  <td>&nbsp;</td>
-  <td>&nbsp;</td>
-  <td>&nbsp;</td>
-  <td>&nbsp;</td>
-  <td>&nbsp;</td>
-  <td>&nbsp;</td>
-  <td>&nbsp;</td>
-  <td>&nbsp;</td>
-  <td>&nbsp;</td>
-  <td>&nbsp;</td>
-  <td>&nbsp;</td>
-  <td>&nbsp;</td>
-  <td>&nbsp;</td>
+{{#if subheader}}
+<td colspan="6">{{title}}</td>
+{{#if reliability}}
+<td>&nbsp;</td>
+{{/if}}
+<td>&nbsp;</td>
+{{#if reliability}}
+<td>&nbsp;</td>
+{{/if}}
+{{#if comparison}}
+<td>&nbsp;</td>
+{{#if reliability}}
+<td>&nbsp;</td>
+{{/if}}
+<td>&nbsp;</td>
+{{#if reliability}}
+<td>&nbsp;</td>
+{{/if}}
+<td>&nbsp;</td>
+{{#if reliability}}
+<td>&nbsp;</td>
+{{/if}}
+<td>&nbsp;</td>
+{{#if reliability}}
+<td>&nbsp;</td>
+{{/if}}
+{{/if}}
+{{else}}
+  {{#if divider}}
+  <td>&nbsp;</td>
+  <td>&nbsp;</td>
+  <td>&nbsp;</td>
+  <td>&nbsp;</td>
+  <td>&nbsp;</td>
+  <td>&nbsp;</td>
+  {{#if reliability}}
+  <td>&nbsp;</td>
+  {{/if}}
+  <td>&nbsp;</td>
+  {{#if reliability}}
+  <td>&nbsp;</td>
+  {{/if}}
+  {{#if comparison}}
+  <td>&nbsp;</td>
+  {{#if reliability}}
```

Comment:
Any way we can simplify these? Instead of wrapping each table cell in an `{{if}}` block, we can do this:

```handlebars
 {{#if reliability}}
 <td>&nbsp;</td>
 <td>&nbsp;</td>
 <td>&nbsp;</td>
 <td>&nbsp;</td>
 <td>&nbsp;</td>
 {{/if}}
```

Reply:
I don't know that it can. As is, it seems that it the cells for reliability could be clustered. However, some of these cells should have the `.cell-border-left` class, as they do in `demographic-table-row.hbs`. With the borders, it's crucial that a certain number of cells appear in a particular order.